File: laramie-importacion-webapi/src/domain/entities/persona.ts
export default class Persona {

    id: number;
	idTipoPersona: number;
	nombrePersona: string;
	idTipoDocumento: number;
	numeroDocumento: string;

	constructor(
        id: number = 0,
		idTipoPersona: number = 0,
		nombrePersona: string = "",
		idTipoDocumento: number = 0,
		numeroDocumento: string = ""
	)
	{
        this.id = id;
		this.idTipoPersona = idTipoPersona;
		this.nombrePersona = nombrePersona;
		this.idTipoDocumento = idTipoDocumento;
		this.numeroDocumento = numeroDocumento;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.idTipoPersona = row.idTipoPersona ?? 0;
		this.nombrePersona = row.nombrePersona ?? "";
		this.idTipoDocumento = row.idTipoDocumento ?? 0;
		this.numeroDocumento = row.numeroDocumento ?? "";
	}

}

File: modules/ingresos_publicos/models/persona.py
from sqlalchemy import Column, BigInteger, String, Boolean

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Persona(Base):
    __tablename__ = "ingresos_publicos_personas"

    id = Column(BigInteger, primary_key=True, autoincrement=True)
    id_tipo_persona = Column(BigInteger, nullable=False)
    id_tipo_documento = Column(BigInteger, nullable=False)
    numero_documento = Column(String(50), nullable=False)
    nombre = Column(String(250), nullable=True)
    apellido = Column(String(250), nullable=True)
    denominacion = Column(String(250), nullable=True)
    activo = Column(Boolean, nullable=False, default=True)
